ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh

```diff
@@ -131,7 +131,7 @@ function pr-prebuild-doxygen-tag(){
                             # Check a comment statement that begins with '/*'.
                             # Note that doxygen does not recognize a comment  statement that start with '/*'.
                             # Let's skip the doxygen tag inspection such as "/**" in case of a single line comment.
-                            if [[ $line =~ "/*" && $line != *"/**"*  && ( $line != *"*/"  || $line =~ "@" ) ]]; then
+                            if [[ $line =~ "/*" && $line != *"/**"*  && ( $line != *"*/"  || $line =~ "@" ) && ( $idx != 1 ) ]]; then
```

Comment:
@again4you PTAL, This PR requires your review. :)

Reply:
Which better? 
* option1: with this PR
* option2: with pr_doxygen_check_level=0 instead of 1
```bash
### configuration file: 
 84 ### Check level of doxygen tag:
 85 # Basic = 0 (@file + @brief)
 86 # Advanced = 1 (Basic + "@author, @bug and functions with ctags")
 87 pr_doxygen_check_level=0
```